rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -1136,9 +969,7 @@ bool opcode_0057(const ScriptArguments& args, const ScriptPlayer player, ScriptV
 	@arg arg2G 
 */
 void opcode_0058(const ScriptArguments& args, ScriptIntGlobal arg1G, ScriptIntGlobal arg2G) {
-	RW_UNIMPLEMENTED_OPCODE(0x0058);
-	RW_UNUSED(arg1G);
-	RW_UNUSED(arg2G);
+	arg1G += arg2G;
```

Comment:
Up until this point the RW_UNUSED(args) did appear before the operation. It seems to change at this point?!